src/main/java/com/internet/shop/dao/jdbc/UserDaoJdbcImpl.java

```
@@ -186,7 +186,7 @@ private Long getRoleByRoleName(String roleName) {
             if (resultSet.next()) {
                 return resultSet.getLong("role_id");
             }
-            return null;
+            return -1L;
```

hardcode is not good
let's throw the exception instead